components/apimgt/org.wso2.carbon.apimgt.internal.service/src/gen/java/org/wso2/carbon/apimgt/internal/service/dto/APIDTO.java

```diff
@@ -192,6 +193,24 @@ public void setStatus(String status) {
     this.status = status;
   }
 
+  /**
+   * Organization of the API.
+   **/
```

Comment:
Shall we update the JavaDoc with the parameter information?

Reply:
Because this is an auto generated file, will not be able to manually edit it.